### PR TITLE
Run examples in both commonjs and esm

### PR DESCRIPTION
**Pull request by chrmod**
`*-example` packages worked with commonjs or esm. With this PR we let user to decide which version to run.

### CODE REVIEW

File: packages/adblocker-electron-example/README.md [hunk 1]
```diff
@@ -46,7 +46,9 @@
 Blocking ads in Electron using [@cliqz/adblocker-electron](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-electron).
 
 ```sh
-$ yarn start
+$ yarn build
+$ yarn start:commonjs
```
Comment (philipp-classen) [inline packages/adblocker-electron-example/README.md:50]: This reads as if you are intended to run both `yarn start:commonjs` and yarn start:esm`. While it should be either-or, I suppose?

File: packages/adblocker-playwright-example/README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 Blocking ads in Playwright using [@cliqz/adblocker-playwright](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-playwright).
 
 ```sh
-$ yarn start
+$ yarn build
```
Comment (philipp-classen) [inline packages/adblocker-playwright-example/README.md:4]: same as above

File: packages/adblocker-puppeteer-example/README.md [hunk 3]
```diff
@@ -46,7 +46,9 @@
 Blocking ads in Puppeteer using [@cliqz/adblocker-puppeteer](https://github.com/ghostery/adblocker/tree/master/packages/adblocker-puppeteer).
 
 ```sh
-$ yarn start
+$ yarn build
```
Comment (philipp-classen) [inline packages/adblocker-puppeteer-example/README.md:49]: same as above

File: packages/adblocker-puppeteer-example/src/index.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { promises as fs } from 'fs';
 
 function getUrlToLoad(): string {
   let url = 'https://www.mangareader.to/';
-  if (process.argv[process.argv.length - 1].endsWith('.ts') === false) {
+  if (process.argv[process.argv.length - 1].endsWith('.js') === false) {
```
Comment (philipp-classen) [inline packages/adblocker-puppeteer-example/src/index.ts:8]: Is this fixing an existing bug? In the changes in this PR alone, I did not notice that files have been renamed.

Comment (chrmod) [reply packages/adblocker-puppeteer-example/src/index.ts:8]: the new system compiles .ts to .js so the script had to be adopted

**Pull request merged**